projects/briey/memtest/makefile

```diff
@@ -5,11 +5,7 @@ BENCH=no
 MULDIV=no
 # "yes" enables tree mem test and tree zeroing
 # run "make tree=no" to disable this
-TREE=yes
-
-ifeq ($(TREE),no)
-	CFLAGS += -DNO_TREE_TEST -DNO_TREE_INIT
-endif
+TREE=no
```

Comment:
Why did we change this to no?

Reply:
I forgot to set it back after testing, changed it back to yes.